ui/pages/onboarding-flow/recovery-phrase/index.scss

```diff
@@ -0,0 +1,110 @@
+.recovery-phrase {
+  &__tips {
+    flex-direction: column;
+
+    ul {
+      list-style: disc;
+      margin-left: 20px;
+    }
+  }
+
+  &__chips {
+    display: grid;
+    grid-template-columns: 160px 160px 160px;
+    justify-items: center;
+    align-items: center;
+    row-gap: 16px;
+
+    &--hidden {
+      filter: blur(5px);
+    }
+  }
+
+  &__secret {
+    position: relative;
+  }
+
+  &__secret-blocker {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    height: 100%;
+    width: 100%;
+    background-color: rgba(0, 0, 0, 0.6);
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+    padding: 8px 0 18px;
+    border-radius: 4px;
+    color: white;
+
+    &--text {
+      margin-top: 32px;
+    }
+  }
+
+  &__chip-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+
+    &__number {
+      font-size: 14px;
```

Comment:
another instance where using typography components or variables should be done.

Reply:
fixed